eng/testing/xunit/xunit.console.targets

```diff
@@ -13,6 +14,7 @@
     <RunScriptCommand>$(RunScriptCommand) $(TargetFileName)</RunScriptCommand>
     <RunScriptCommand>$(RunScriptCommand) -xml $(TestResultsName)</RunScriptCommand>
     <RunScriptCommand>$(RunScriptCommand) -nologo</RunScriptCommand>
+    <RunScriptCommand>$(RunScriptCommand) -nomethod @(SkipTests, '-nomethod')</RunScriptCommand>
```

Comment:
Do we need to add a condition here like `'@(SkipTests)' != ''`? Otherwise won't a `-nomethod` option get added?

Reply:
I think this will not work if someone runs `dotnet test` locally. I think we should find a way that it works both with `CI` and `dotnet test`.